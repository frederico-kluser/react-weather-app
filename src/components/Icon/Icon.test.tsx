import renderer from 'react-test-renderer';
import Image from '.';

test('Image Renderer', () => {
  const component = renderer.create(
    <Image icon="Heavy snow" />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});