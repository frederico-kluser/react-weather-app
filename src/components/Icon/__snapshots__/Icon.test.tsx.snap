// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Renderer 1`] = `
<img
  className="sc-bdvvtL kLVEJL"
  src="sunSnow.png"
/>
`;
